Add PersonEndpoint and Run-method at Application

komXXme opened the pull request:
Change Dependencies

Commit:
Add PersonEndpoint and Run-method at Application
Change Dependencies

vladimirlogachev reviewed (changes requested):
Nice job! But there are some things to clean up.

VladimirLogachev commented:
Also, please add 
`- OpenJDK 11`
to the `## Prerequisites` section of the readme.

Commit:
Add ZIO config and Dependencies
add ZLayer and default properties at application.conf
add name and description to PersonEndpoint
Fix README.md
Fix imports
Delete helloEndpoint

Commit:
Merge remote-tracking branch 'origin/develop' into tapir-endpoint

# Conflicts:
#	src/main/g8/$name;format="normalize"$/src/main/resources/application.conf
#	src/main/g8/$name;format="normalize"$/src/main/scala/$package$/Application.scala

Commit:
Resolve conflicts

VladimirLogachev commented on src/main/g8/$name;format="normalize"$/src/main/scala/$package$/Application.scala:
```diff
@@ -1,15 +1,51 @@
+      _ <- Console.printLine(
+        s"Go to http://localhost:\${port}/docs to open SwaggerUI. Press ENTER key to exit."
+      )
+      _ <- Console.readLine
```
Why do you think we need the server to go down with a single keystroke? Maybe remove it?

Commit:
Fix naming HttpServerConfig at application.conf
delete commented Dependencies
